src/redux/actions/posts.js

```diff
@@ -0,0 +1,27 @@
+import { database } from 'firebase'
+
+export const getPostsRequested = () => dispatch => {
+  dispatch({ type: 'GET_POSTS_REQUESTED' })
+}
+
+export const getPostsRejected = () => dispatch => {
```

Comment:
you don't need to thunk this one. I'm surprised the code worked like this. 

but if you remove `=> dispatch` and just return the action object. It should continue to work.